docs/setup/install/start-es-and-enroll.asciidoc

```diff
@@ -0,0 +1,21 @@
+==== Start {es} and generate an enrollment token for {kib}
+++++
+<titleabbrev>Generate an enrollment token</titleabbrev>
+++++
+
+When you start {es} for the first time, the following security configuration
+occurs automatically:
+
+* {ref}/configuring-stack-security.html#stack-security-certificates[Certificates and keys] for TLS are
+generated for the transport and HTTP layers.
+* The TLS configuration settings are written to `elasticsearch.yml`.
+* A password is generated for the `elastic` user.
+* An enrollment token is generated for {kib}.
+
+You can then start {kib} and enter the enrollment token, which is valid for 30
+minutes. This token automatically applies the security settings from your {es} 
+cluster, authenticates to {es} with the built-in `kibana` service account, and
```

Comment:
Maybe it would be good to link to the [service accounts docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/service-accounts.html) here?

```suggestion
cluster, authenticates to {es} with the built-in `elastic/kibana` {ref}/service-accounts.html[service account], and
```

Reply:
Is this a bit too much technical detail for a getting started guide? 

I think its enough to say: 

> You can then start Kibana and enter the enrollment token to securely connect Kibana with Elasticsearch.
> 
> The enrollment token is valid for 30 minutes. To generate a new token, run the elasticsearch-create-enrollment-token tool.

Reply:
I tend to agree with @thomheymann that simplicity is probably more useful in this context.